common/configuration/asm_test.go

```diff
@@ -0,0 +1,127 @@
+package configuration
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"testing"
+
+	"github.com/TBD54566975/ftl/internal/log"
+
+	"github.com/alecthomas/assert/v2"
+	. "github.com/alecthomas/types/optional"
+)
+
+func localstack() *ASM[Secrets] {
```

Comment:
Pass in a `*testing.T` so you can use normal test assertions rather than panicking.